app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,50 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+
+/* universal */
+html, body {
+  height: 100%;
+}
+body {
+  padding-top: 60px;
+  background: url('cobbles.jpg')no-repeat center center fixed;
+  background-size: cover;
+}
+
+
+
+/* header stuff */
+#logo {
+  float: left;
+  margin-right: 10px;
+  font-size: 1.7em;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.